### PR TITLE
doc: add steps for doing a release

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2474,6 +2474,27 @@ Possible values:
...
 
+## Doing a release
+
+To do a release complete the following steps:
+
+* Look at the list of changes - https://github.com/nodejs/uvwasi/commits/main.
+* Put together a list of notable changes.
+  See https://github.com/nodejs/uvwasi/releases/tag/v0.0.14
+  or any of the other releases for example. Use that list in the release commit,
+  the github release, and the PR to update uvwasi in node (or any other
```
Comment (cjihrig) [inline README.md:2485]:
```suggestion
  the GitHub release, and the PR to update uvwasi in Node.js (or any other
```

File: README.md [hunk 2]
```diff
@@ -2474,6 +2474,27 @@ Possible values:
...
+  projects where you updat it)
```
Comment (cjihrig) [inline README.md:2486]:
```suggestion
  projects where you update it)
```

File: README.md [hunk 3]
```diff
@@ -2474,6 +2474,27 @@ Possible values:
...
+* Create a release commit. this should just involve changing one line and
```
Comment (cjihrig) [inline README.md:2487]:
```suggestion
* Create a release commit. This should just involve changing one line and
```

File: README.md [hunk 4]
```diff
@@ -2474,6 +2474,27 @@ Possible values:
...
+  adding the notable changes. See
+  https://github.com/nodejs/uvwasi/commit/6ad5fc996420d0e4e75983ce3deb65f327321f33
+  as an example.
+* PR the release commit. once it lands, create a github release with
```
Comment (cjihrig) [inline README.md:2491]:
```suggestion
* PR the release commit. Once it lands, create a GitHub release with
```

File: README.md [hunk 5]
```diff
@@ -2474,6 +2474,27 @@ Possible values:
...
+  the same notable changes. When doing the GitHub release you will need to select
+  `Choose a tag` and type in the new tag. That should result in 
+  ` Create new tag: vX.Y.Z on publish` where X.Y.Z matches the tag you specificed.
```
Comment (cjihrig) [inline README.md:2494]:
```suggestion
  ` Create new tag: vX.Y.Z on publish` where vX.Y.Z matches the tag you specified.
```

File: README.md [hunk 6]
```diff
@@ -2474,6 +2474,27 @@ Possible values:
...
+* Update uvwasi in node or any projects you want to update - there are several
```
Comment (cjihrig) [inline README.md:2495]:
```suggestion
* Update uvwasi in Node.js or any projects you want to update - there are several
```

**Comment by mhdawson** [comment]
who knew I could get so many typos in a short paragraph. @cjihrig thanks for catching all of those.

**Review by cjihrig — APPROVED** [review]

**Review by phated — APPROVED** [review]
lgtm

**Pull request merged**